mautrix_facebook/example-config.yaml

```diff
@@ -109,6 +109,9 @@ bridge:
     # Localpart template of MXIDs for Facebook users.
     # {userid} is replaced with the user ID of the Facebook user.
     username_template: "facebook_{userid}"
+    # Whether or not to enable creating a space per user and inviting the user
+    # (as well as all of the puppets) to that space.
+    enable_space_per_user: true
```

Comment:
Probably should default to false

Reply:
Changed to false in the current revision + renamed it to conform with LinkedIn :)